### PR TITLE
Adds default values for DateAndLocation page for HHG/PPM move and tes…

### PR DESCRIPTION
**Pull request by chrisrcoles**
## Description

This PR updates the Add Date and Location page for **_HHG/PPM combo_** by adding default values to the page upon initial load. 

## Reviewer Notes

Currently, test is failing given `/^\/moves\/[^/]+\/hhg-ppm-size/` has been implemented before `/^\/moves\/[^/]+\/hhg-ppm-weight/`. Once [feature](https://www.pivotaltracker.com/story/show/162018003) has been implemented, the test will be passing. 

## Setup

1. `make server_run`
2. `make client_run`
3. Complete HHG + PPM combo move flow

## Code Review Verification Steps
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/161767800) for this change

### CODE REVIEW

File: src/scenes/Moves/Ppm/DateAndLocation.jsx [hunk 1]
```diff
@@ -187,13 +188,31 @@ function mapStateToProps(state) {
...
+
+  const currentShipment = selectShipment(state, getCurrentShipmentID(state));
+  const addresses = state.entities.addresses;
+
+  if (props.isHHGPPMComboMove) {
+    props.initialValues = {
+      ...props.initialValues,
+      planned_move_date: currentOrders.issue_date,
+      // defaults to SM's destination address, if none, uses destination duty station zip
+      destination_postal_code:
```
Comment (sarboc) [inline src/scenes/Moves/Ppm/DateAndLocation.jsx]: Can you move this logic into a selector? We're getting a lot of business logic in `mapStateToProps`, and it's both cleaner and more easily tested to move it into a selector.

Comment (chrisrcoles) [reply src/scenes/Moves/Ppm/DateAndLocation.jsx]: NP - I've moved it to `scenes/Moves/Ppm/ducks.js` as the function `getDestinationPostalCode`. Let me know if that placement keeps in line with the conventions that we've set for the codebase and using selectors. Thx.

**Comment by kahlouie** [comment]
I think I've seen some conflict changes with my pr that landed yesterday: specifically https://github.com/transcom/mymove/blob/master/cypress/integration/mymove/hhgPPM.js#L7-L9.

I'll review it after that's fixed up.

File: cypress/integration/mymove/hhgPPM.js [hunk 2]
```diff
@@ -105,9 +101,9 @@ function serviceMemberCanReviewMoveSummary() {
     const text = $div.text();
     expect(text).to.include('Shipment - You move your stuff (PPM)');
     expect(text).to.include('Move Date: 09/02/2018');
-    expect(text).to.include('Pickup ZIP Code:  80913');
-    expect(text).to.include('Delivery ZIP Code:  76127');
-    expect(text).not.to.include('Storage: Not requested');
+    expect(text).to.include('Pickup ZIP Code:  90210');
+    expect(text).to.include('Delivery ZIP Code:  50309');
+    expect(text).to.include('Storage: Not requested');
```
Comment (kahlouie) [inline cypress/integration/mymove/hhgPPM.js]: I think we don't want to test that this line exists. We've removed that for hhgPPM moves with https://github.com/transcom/mymove/pull/1367

**Pull request merged**